### PR TITLE
Fix various typos not caught by codespell

### PR DESCRIPTION
**Pull request by luzpaz**

### CODE REVIEW

**Comment by luzpaz** [comment]
Please review [966b2d4](https://github.com/godotengine/godot/pull/63281/commits/966b2d40ea08c592dc12cff252dbc024969dcd30) closely

File: core/input/gamecontrollerdb.txt [hunk 1]
```diff
@@ -439,7 +439,7 @@
...
 030000004c0500003713000000000000,PlayStation Vita,a:b1,b:b2,back:b8,dpdown:b13,dpleft:b15,dpright:b14,dpup:b12,leftshoulder:b4,leftx:a0,lefty:a1,rightshoulder:b5,rightx:a3,righty:a4,start:b9,x:b0,y:b3,platform:Windows,
 03000000d62000006dca000000000000,PowerA Pro Ex,a:b1,b:b2,back:b8,dpdown:h0.4,dpleft:h0.8,dpright:h0.2,dpup:h0.1,guide:b12,leftshoulder:b4,leftstick:b10,lefttrigger:b6,leftx:a0,lefty:a1,rightshoulder:b5,rightstick:b11,righttrigger:b7,rightx:a2,righty:a3,start:b9,x:b0,y:b3,platform:Windows,
-0300000062060000d570000000000000,PowerA PS3 Contoller,a:b2,b:b1,back:b8,dpdown:h0.4,dpleft:h0.8,dpright:h0.2,dpup:h0.1,guide:b12,leftshoulder:b4,leftstick:b10,lefttrigger:b6,leftx:a0,lefty:a1,rightshoulder:b5,rightstick:b11,righttrigger:b7,rightx:a2,righty:a3,start:b9,x:b0,y:b3,platform:Windows,
+0300000062060000d570000000000000,PowerA PS3 Controller,a:b2,b:b1,back:b8,dpdown:h0.4,dpleft:h0.8,dpright:h0.2,dpup:h0.1,guide:b12,leftshoulder:b4,leftstick:b10,lefttrigger:b6,leftx:a0,lefty:a1,rightshoulder:b5,rightstick:b11,righttrigger:b7,rightx:a2,righty:a3,start:b9,x:b0,y:b3,platform:Windows,
```
Comment (akien-mga) [inline core/input/gamecontrollerdb.txt:442]: This should be fixed upstream: https://github.com/gabomdq/SDL_GameControllerDB

**Comment by akien-mga** [comment]
You say "typos **not caught** by codespell" but this looks like the typical class of typos that codespell does find. How did you find them?

**Comment by luzpaz** [comment]
Running a HEAD version of codespell with multiple dictionaries (+ external dictionaries)

**Review by akien-mga — APPROVED** [review]
Looks good. Commits should be squashed.

File: core/input/gamecontrollerdb.txt (hunk 1 above)
Comment (luzpaz) [reply core/input/gamecontrollerdb.txt]: Reverted and filed a PR upstream https://github.com/gabomdq/SDL_GameControllerDB/pull/600

**Comment by luzpaz** [comment]
> Commits should be squashed.  

Done

**Review by akien-mga — APPROVED** [review]

**Comment by akien-mga** [comment]
Thanks!

**Pull request merged**